SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs

```diff
@@ -341,6 +341,73 @@ public static void Main(string[] args)
                     "-archive=" + Quote(Path.Combine(outputDir, $"UnrealWorker@{assemblyPlatform}.zip"))
                 });
             }
+            else if (gameName == baseGameName + "Client")
+            {
+                Common.WriteHeading(" > Building client.");
+                Common.RunRedirected(runUATBat, new[]
+                {
+                    "BuildCookRun",
+                    noCompile ? "-nobuild" : "-build",
+                    noCompile ? "-nocompile" : "-compile",
+                    "-project=" + Quote(projectFile),
+                    "-noP4",
+                    "-clientconfig=" + configuration,
+                    "-serverconfig=" + configuration,
+                    "-utf8output",
+                    "-cook",
+                    "-stage",
+                    "-package",
+                    "-unversioned",
+                    "-compressed",
+                    "-stagingdirectory=" + Quote(stagingDir),
+                    "-stdout",
+                    "-FORCELOGFLUSH",
+                    "-CrashForUAT",
+                    "-unattended",
+                    "-fileopenlog",
+                    "-SkipCookingEditorContent",
+                    "-client",
+                    "-noserver",
+                    "-platform=" + platform,
+                    "-targetplatform=" + platform,
+                    additionalUATArgs
+                });
+
+                var windowsClientPath = Path.Combine(stagingDir, "WindowsClient");
+
+                if (additionalUATArgs.Contains("-pak"))
+                {
+                    Console.WriteLine("Cannot force bSpatialNetworking with -pak argument.");
+                }
+                else
+                {
+                    ForceSpatialNetworkingInConfig(windowsClientPath, baseGameName);
+                }
```

Comment:
Move this out into a function (especially because this is already copy-pasted for all targets).